homeassistant/components/yolink/binary_sensor.py

```diff
@@ -124,6 +124,7 @@ def __init__(
     ) -> None:
         """Init YoLink Sensor."""
         super().__init__(config_entry, coordinator)
+        self.coordinator = coordinator
```

Comment:
```suggestion
```
`YoLinkBinarySensorEntity` extends from `YoLinkEntity` and `CoordinatorEntity`, This line is not needed